.github/workflows/get-pass-secrets.yaml

```diff
@@ -0,0 +1,49 @@
+name: Get and pass secrets
+
+on:
+  push:
+    branches:
+      - master
+      - feature/pre-commit
+
+jobs:
+  get-secrets:
+    runs-on: ubuntu-latest
+    outputs:
+      username: ${{ steps.read-secrets.outputs.mail_username }}
+      password: ${{ steps.read-secrets.outputs.mail_password }}
+      to: ${{ steps.read-secrets.outputs.mail_to }}
+      server_address: ${{ steps.read-secrets.outputs.mail_server_address }}
+      server_port: ${{ steps.read-secrets.outputs.mail_server_port }}
+    steps:
+      - name: Checkout code from the repository
+        uses: actions/checkout@v4
+
+      - name: Unlock secrets with git-crypt
+        if: always()
+        uses: sliteteam/github-action-git-crypt-unlock@1.2.0
+        env:
+          GIT_CRYPT_KEY: ${{ secrets.GIT_CRYPT_KEY }}
+
+      - name: Read secrets from secrets.txt
+        id: read-secrets
+        if: always()
+        run: |
+          cat secrets.txt >> "$GITHUB_OUTPUT"
+        working-directory: .
```

Comment:
Is it really needed? Let's specify it as default params?

Reply:
Fixed